library/core/src/slice/iter.rs

```diff
@@ -3174,7 +3220,11 @@ where
             let mut len = 1;
             let mut iter = self.slice.windows(2);
             while let Some([l, r]) = iter.next() {
-                if (self.predicate)(l, r) { len += 1 } else { break }
+                if (self.predicate)(l, r) {
+                    len += 1
+                } else {
+                    break;
+                }
```

Comment:
accidental formatting changes? they don't look relevant to this PR

Reply:
Gosh dangit, I keep leaving my editor's autoformatter on and for whatever reason it keeps misformatting this file.